mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp

```diff
@@ -394,137 +395,179 @@ PopulateParamsAccel::obtainTuningParameters(RockGemmWrapperInterface op,
-  {64, 64, 8, 32, 32, 4, 1, true, true},
-  {64, 64, 8, 16, 64, 8, 1, true, true},
-  {64, 64, 8, 16, 64, 4, 1, true, true},
-  {64, 64, 4, 32, 32, 8, 1, true, true},
-  {64, 64, 4, 16, 64, 8, 1, true, true},
-  {64, 64, 4, 16, 64, 4, 1, true, true},
-  {32, 128, 4, 32, 32, 8, 1, true, true},
-  {32, 64, 8, 16, 64, 8, 1, true, true},
-  {32, 64, 4, 32, 64, 4, 1, true, true},
-  {32, 64, 4, 32, 32, 8, 1, true, true},
-  {32, 64, 4, 16, 64, 8, 1, true, true},
-  {32, 64, 2, 32, 64, 8, 1, true, true},
-  {32, 32, 8, 16, 16, 8, 1, true, true},
-  {32, 32, 4, 16, 16, 4, 1, true, true},
-  {16, 64, 8, 16, 16, 8, 1, true, true},
-  {16, 64, 4, 8, 64, 4, 1, true, true},
-  {16, 64 ,4, 16, 16, 8, 1, true, true},
-  {16, 32, 4, 16, 16, 8, 1, true, true},
-  {16, 32, 4, 16, 16, 4, 1, true, true},
-  {16, 16, 4, 16, 16, 8, 1, true, true},
-  {8, 64, 8, 8, 64, 8, 1, true, true}
+  // M/block N/block K/block M/wave mnkPerXdl kPack splitKFactor forceUnroll bCopyMore
+  {128, 256, 8, 64, 3208, 4, 1, true, true},
+  {128, 256, 4, 64, 3208, 8, 1, true, true},
+  {128, 256, 2, 64, 3208, 8, 1, true, true},
+  {128, 256, 2, 64, 3208, 4, 1, true, true},
+  {128, 128, 4, 64, 3208, 8, 1, true, true},
+  {128, 128, 8, 64, 3208, 1, 1, true, true},
+  {128, 128, 8, 64, 3208, 1, 1, true, true},
+  {128, 128, 8, 64, 3208, 4, 1, true, true},
+  {128, 128, 4, 64, 3208, 8, 1, true, true},
+  {128, 128, 4, 64, 3208, 4, 1, true, true},
+  {128, 128, 2, 64, 3208, 8, 1, true, true},
+  {128, 128, 2, 64, 3208, 8, 1, true, true},
+  {64, 256, 2, 64, 3208, 8, 1, true, true},
+  {64, 128, 8, 32, 3208, 4, 1, true, true},
+  {64, 128, 4, 32, 3208, 4, 1, true, true},
+  {64, 128, 4, 32, 3208, 8, 1, true, true},
+  {64, 128, 2, 64, 3208, 8, 1, true, true},
+  {64, 128, 2, 32, 3208, 8, 1, true, true},
+  // just keeping one for bf16 for gfx908
+  {64, 64, 8, 32, 3204, 8, 1, true, true},
+  {64, 64, 8, 32, 3208, 8, 1, true, true},
+  {64, 64, 8, 32, 3208, 4, 1, true, true},
+  {64, 64, 8, 16, 1616, 8, 1, true, true},
+  {64, 64, 8, 16, 1616, 4, 1, true, true},
+  {64, 64, 4, 32, 3208, 8, 1, true, true},
+  {64, 64, 4, 16, 1616, 8, 1, true, true},
+  {64, 64, 4, 16, 1616, 4, 1, true, true},
+  {32, 128, 4, 32, 3208, 8, 1, true, true},
+  {32, 64, 8, 16, 1616, 8, 1, true, true},
+  {32, 64, 4, 32, 3208, 4, 1, true, true},
+  {32, 64, 4, 32, 3208, 8, 1, true, true},
+  {32, 64, 4, 16, 1616, 8, 1, true, true},
+  {32, 64, 2, 32, 3208, 8, 1, true, true},
+  {32, 32, 8, 16, 1616, 8, 1, true, true},
+  {32, 32, 4, 16, 1616, 4, 1, true, true},
+  {16, 64, 8, 16, 1616, 8, 1, true, true},
+  {16, 64, 4, 8, 404, 4, 1, true, true},
+  {16, 64 ,4, 16, 1616, 8, 1, true, true},
+  {16, 32, 4, 16, 1616, 8, 1, true, true},
+  {16, 32, 4, 16, 1616, 4, 1, true, true},
+  {16, 16, 4, 16, 1616, 8, 1, true, true},
+  {8, 64, 8, 8, 404, 8, 1, true, true}
 };
 
 const InitParamsAccel
 PopulateParamsXDL::initParametersForward8Bit[
   PopulateParamsXDL::nInitParametersForward8Bit] = {
```

Comment:
Do we have a justification for making the int8 quick tuning list twice as long? Paul's really not going lo like it and for good reason.

Reply:
Its not twice as long practically.
only half of it is valid for pre-MI300 and half of it valid for post-MI300.